.github/ISSUE_TEMPLATE/Bug_report.md

```diff
@@ -0,0 +1,32 @@
+---
+name: Bug report
+about: Create a report to help us improve our libraries
+
+---
+
+<!--
+  Thanks for participating in this project! We will try to answer as soon as possible 🙏
+-->
+
+**Describe the bug 🐛**
+A clear and concise description of what the bug is.
+
+**To Reproduce 🔍**

```

Comment:
As it's a library, we can also add a _code snippet_ revealing the bug as a required information for reproduction.